src/app/lists/ListDataview.tsx

```diff
@@ -36,8 +36,9 @@ interface QuantityAndNotes {
     notes: string | null;
 }
 
-interface Props {
-    data: Schema["lists"][];
+interface ListDataViewProps {
+    listOfIngridients: Schema["lists"][];
+    setData: React.Dispatch<React.SetStateAction<Schema["lists"][]>>;
```

Comment:
should we also rename this, like `setListOfIngredients`?

Reply:
Renamed